UnitsNet.Tests/CustomCode/LevelTests.cs

```diff
@@ -42,5 +42,19 @@ public void InvalidReference_ExpectArgumentOutOfRangeException(double quantity,
             // reference can't be zero or less than zero if quantity is postive.
             Assert.Throws<ArgumentOutOfRangeException>(() => new Level(quantity, reference));
         }
+
+        [Theory]
+        [InlineData(-1.0, -1.0)]
+        [InlineData(-2.0, -0.5)]
+        [InlineData(0.0, 0.0)]
+        [InlineData(1.0, 1.0)]
+        [InlineData(2.0, 0.5)]
+        public static void InverseTest(double value, double expected)
```

Comment:
I see you already copied this test to `UnitsNet.Tests/CustomCode/ReciprocalLengthTests.cs`, so you can remove this.

Reply:
I thought this test was part of the LengthTests so an optical illusion happened for me :). Made this test a part of the LengthTests.